packages/shared/routes/dashboard/wallet/views/Send.svelte

```diff
@@ -259,7 +256,12 @@
     {#if !$isTransferring}
         <div class="flex flex-row justify-between px-2">
             <Button secondary classes="-mx-2 w-1/2" onClick={() => handleBackClick()}>{locale('actions.cancel')}</Button>
-            <Button classes="-mx-2 w-1/2" onClick={() => handleSendClick()}>{locale('actions.send')}</Button>
+            <Button
+                classes="-mx-2 w-1/2"
+                onClick={() => handleSendClick()}
+                disabled={selectedSendType === SEND_TYPE.INTERNAL && !to}>
```

Comment:
what about using error handlers here instead of disabling the button?

Reply:
Done